skins/oasis/css/modules/ContentReview.scss

```diff
@@ -48,8 +48,28 @@ $content-review-color-unsubmitted: #ffad00;
 
 .content-review-status-unsubmitted {
 	border-color: $content-review-color-unsubmitted;
+	animation: submitReminder 3s infinite;
 }
 
 .content-review-status-awaiting {
 	border-color: $content-review-color-awaiting;
 }
+
+/* Animation */
+@keyframes submitReminder {
```

Comment:
0% margin-left: 0;
100% margin-left: 10px;

and when invoking
animation: submitReminder .75s ease 1.5s infinite;

that should be the same right?